bundles/PersonalizationBundle/src/Targeting/EventListener/TargetingEventListener.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\PersonalizationBundle\Targeting\EventListener;
+
+use Pimcore\Bundle\CoreBundle\EventListener\Traits\EnabledTrait;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+class TargetingEventListener implements EventSubscriberInterface
+{
+
+    use EnabledTrait;
+
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            KernelEvents::REQUEST => ['onKernelRequest']
+        ];
+    }
+
+    public function onKernelRequest(RequestEvent $event): void
```

Comment:
It's moved to a diffetent name because the TargetingListener.php would be loaded only if the personalization bundle is enabled. To avoid the confusion, the file is renamed to TargetingEnabledListener in [041b52d](https://github.com/pimcore/pimcore/pull/14462/commits/041b52d81d5e8199b35a579ac2f31ec90e74cdc8)